tests/cassettes/test_user_update.yaml

```diff
@@ -85,31 +85,31 @@ interactions:
       Content-Type:
       - application/json
       authorization:
-      - REDACTED
+      - <REDACTED>
       user-agent:
-      - REDACTED
+      - <REDACTED>
       x-client-user-agent:
-      - REDACTED
-    method: PUT
-    uri: https://api.easypost.com/v2/users/user_54356a6b96c940d8913961a3c7b909f2
+      - <REDACTED>
+    method: PATCH
+    uri: https://api.easypost.com/v2/users/user_2cfd34d1261a4a9b93d56c313a9a824b
   response:
     body:
-      string: '{"id": "user_54356a6b96c940d8913961a3c7b909f2", "object": "User", "parent_id":
-        null, "name": "EasyPost Python Client Library Tests", "phone_number": "REDACTED",
-        "verified": true, "created_at": "2017-03-01T03:01:34Z", "balance": "0.00000",
-        "price_per_shipment": "0.00000", "recharge_amount": null, "secondary_recharge_amount":
-        null, "recharge_threshold": null, "has_billing_method": null, "cc_fee_rate":
-        "0.03", "default_insurance_amount": null, "insurance_fee_rate": "0.01", "insurance_fee_minimum":
-        "1.00", "email": "REDACTED", "children": ["REDACTED"]}'
+      string: '{"id": "user_2cfd34d1261a4a9b93d56c313a9a824b", "object": "User", "parent_id":
+        null, "name": "Ryan Nguyen", "phone_number": "<REDACTED>", "verified": true,
```

Comment:
Nit: Ryan ran this with his personal API keys. No problem, it will get overridden eventually in the future when we re-record cassettes

Reply:
Eh, this is true of any 3rd party contributor. With censoring in place, it should be safe for anyone to run any of the tests.